src/liquid/components/ld-stepper/ld-step/test/ld-step.spec.tsx

```diff
@@ -306,4 +322,14 @@ describe('ld-step', () => {
 
     expect(button.focus).toHaveBeenCalled()
   })
+
+  it('does not throw when setting inner focus on nothing', async () => {
+    const component = new LdStep()
+    await component.focusInner()
+  })
```

Comment:
Is this really worth testing, as the component is not meant to be used this way and does not work when being used like that anyway?